### ISSUE TITLE
[Suggestion] Virtual Farm Support

### ISSUE THREAD

**Issue by Manuepolis23** [Arcaratus/VirtualMachines#2]
Can you add support for Mystical Agriculture seeds?

Thanks and keep up with good work!

**Comment by Arcaratus** [comment]
Yes, mod interactions are planned for future releases, I just haven't gotten around to them yet.
